src/sbd-inquisitor.c

```diff
@@ -1307,37 +1303,50 @@ int main(int argc, char **argv, char **envp)
         } else if (strcmp(argv[optind], "ping") == 0) {
             exit_status = ping_via_slots(argv[optind + 1], servants_leader);
 
+        } else {
```

Comment:
thought we'd move that outside the with-disk-section right at the end.
```
} else if (strcmp(argv[optind], "ping") == 0) {
            exit_status = ping_via_slots(argv[optind + 1], servants_leader);
} else
#endif

if (strcmp(argv[optind], "query-watchdog") == 0) {
...
} else {
    exit_status = -2;
}

```

Reply:
Ah ok, and split the watch option from it right ?

Reply:
Ok I've pushed the commit with the white space fix to soon.